CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 
 Improvements:
 * [OLMIS-6478](https://openlmis.atlassian.net/browse/OLMIS-6478): Made Superset Server URL Configurable
+* [OLMIS-6504](https://openlmis.atlassian.net/browse/OLMIS-6504): Move AO-297 implementation to the Core project
```

Comment:
please give a better description of those changes

Reply:
Improved